staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -41,6 +41,12 @@ const (
 	// pluggable output backends and an audit policy specifying how different requests should be
 	// audited.
 	AdvancedAuditing utilfeature.Feature = "AdvancedAuditing"
+
+	// owner: @ilackams
+	// alpha: v1.7
+	//
+	// Enables compression of REST responses (GET and LIST only)
+	EnableAPIResponseCompression utilfeature.Feature = "EnableAPIResponseCompression"
```

Comment:
@liggitt it was until @deads2k told me to rename it to `EnableAPIResponseCompression`

what's the final word on this?